modules/library/metadata/src/main/java/org/geotools/util/Utilities.java

```diff
@@ -755,4 +760,12 @@ public static void ensureArgumentNonNull(final String name, final Object object)
                 Function.identity(),
                 (first, second) -> second);
     }
+
+    /**
+     * @param fileName
```

Comment:
If at all possible, write a description of what any new public and protected  method and class does, the params documentation is of secondary importance (nice to have) but there is not point in having them there, and empty (``@param fileName``) though CheckStyle won't complain (yet). Other tools do, for example lgtm.com